runtime-modules/working-group/src/types.rs

```diff
@@ -196,6 +197,9 @@ pub struct Worker<AccountId, RewardRelationshipId, StakeId, BlockNumber, MemberI
 
     /// When set, describes role stake of the worker/lead.
     pub role_stake_profile: Option<RoleStakeProfile<StakeId, BlockNumber>>,
+
+    /// Raw storage field
+    pub storage: Vec<u8>,
```

Comment:
One issue to address is that with the addition of this field, we need to migrate the existing workers during runtime upgrade.

One approach is to implement a custom `Decode` trait for `struct Worker` that can deal with missing field.
Alternatively we can store this `storage` value outside of this struct in a separate state storage map:

```rust
decl_storage! {
   ...
   WorkerStorage map WorkerId => Vec<u8>;
}
```

Reply:
I chose to introduce an additional storage map :) https://github.com/Joystream/joystream/pull/2180/commits/956934ab2be4f0f3aa3fbdaac23813db3159cfd5